src/renderer/components/+preferences/preferences.tsx

```diff
@@ -25,6 +25,7 @@ export class Preferences extends React.Component {
   @observable helmRepos: HelmRepo[] = [];
   @observable helmAddedRepos = observable.map<string, HelmRepo>();
   @observable httpProxy = userStore.preferences.httpsProxy || "";
+  @observable shell = userStore.preferences.shell || "";
```

Comment:
Why do we need to copy value from `userStore`? Only because of saving on `blur`?
Other things like `preferences.colorTheme` done by updating store directly, e.g. `preferences.tsx:128`.

Reply:
Yeah, I guess it's that we don't want to update store on every key press.

Reply:
Should we then delay updating other preferences as well?

``` 
onChange={({ value }: SelectOption) => preferences.colorTheme = value}
```